terragrunt/env/production/efs/terragrunt.hcl

```diff
@@ -0,0 +1,24 @@
+terraform {
+  source = "../../../aws//efs"
+}
+
+dependencies {
+    paths = ["../vpc"]
+}
+
+dependency "vpc" {
+  config_path = "../vpc"
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs = {
```

Comment:
Were you able to get the `mock_outputs_merge_strategy_with_state` attribute to work or was it still causing errors locally?
https://github.com/cds-snc/scan-files/blob/ae0c4a41f49f245224f40678641f6489bb83acdb/terragrunt/env/production/alarms/terragrunt.hcl#L13

Reply:
So this is the error I have been getting locally:

`Unsupported argument; An argument named "mock_outputs_merge_strategy_with_state" is not expected here.`

Reply:
It's possibly related to the version of Terragrunt you have installed locally.  I know it's a newer attribute.